### ISSUE TITLE
Encryption wont unlock!

### ISSUE THREAD

**Issue by jdomsky** [davorcc/DavorCoin#3]
im in an emergency situation. I have downloaded the Windows Qt Staking Wallet, and I sent my 570 Davorcoin in to the wallet. I backed up the wallet, and I encrypted it with my passphrase. Now I am trying to unlock the wallet and it will not unlock, and i know for 110% absolute fact that im putting in the correct passphrase. Im not a beginner by any means, and I write everything down as I go. It will just not unlock, and I dont know what to do. Please help. Thanks.

**Comment by davorcc** [comment]
Dear jdomsky, 

If you have entered the passphrase correctly, the wallet will be unlocked, there is no exception. Please try to remember the passphrase you entered before, one mistype character could render the passphrase unusable.

Best regards,

**Comment by jdomsky** [comment]
Ive been emailing with Julien, and he wont listen to me either! Like I said in my first comment, I am without a doubt, 10000%, for a FACT putting in the CORRECT passphrase!!!! number one, I wrote it down, and number two, I remember it, and its not unlocking!!! Ive been working on computers for 15 years, ive been in crypto for 2 years, and ive used many many wallets, and im not a beginner. Julien wouldnt listen to me, im telling you guys....I am absoutely putting in the correct passphrase!!! It will not unlock!!!! I was so excited about the Davorcoin ICO, it was my favorite, and I felt so lucky to have bought my 570 Dav because it was so hard to buy as fast as it was selling out. And now theyre just going to waste because of something that is not my fault....it makes me sick....like literally. This would have been life changing for me....and now its just going to waste. thanks for replying,

**Comment by davorcc** [comment]
Dear jdomsky,

I am so sorry to hear of your trouble. This is really a situation which is out of our reach. Did you download the QT wallet from our website, because there were some malicious wallet built from our sources code posted on Bitcointalk?

If you are 100% sure the QT wallet is fine, I suggest you to find Bitcoin wallet unlocking services on internet. I don't have my personal experience on this so I can not recommend any service for you.

I hope you could find a way to recover the lost coins.

Best regards,

**Comment by jdomsky** [comment]
how is this a situation that is out of your reach?? its your wallet, and you have every opportunity to do the right thing. Yes, of course I downloaded from your website, I know better then to download anything from outside sources because of that very fact that people create malicious things. and Julien told me the same thing, to hire a service that can brute force it. so you guys are telling me that everyones wallets can simply be compromised at anytime? Im not blaming you guys, and saying you guys have done something horrible.....because I do know that sometimes things just happen, and its nobodies fault. But I am really upset that Julien continues to not listen to me, and he keeps telling me to enter in the correct password, when i am.....im not a beginner, and ive created a million passwords in my life. thats like doing 1+1=2. Its a password, not rocket science. But i always write down my passwords just incase, so I did write it down, and I do remember it too! Even some type of credit to my account would make me feel like you guys have a little honor in you. You do have the power to do that, and that is well within your reach!!

**Comment by davorcc** [comment]
Hi jdomsky,

DavorCoin-Qt is a open-sourced blockchain wallet, all of your private keys are stored and encrypted in the wallet, we do not have any privilege to access to that information. This wallet is as secure as Bitcoin-Qt wallet, and it is used by hundred of thousands people in the world.

Please be kind and do your own research.

Best regards,

**Comment by jdomsky** [comment]
Im not sure what that message meant, ive never said anything bad about the wallet, even with this situation. Like i said im not blaming you guys, or saying anything bad, because sometimes crazy things just happen and theres not always an explanation. But one thing I can explain is that i am without a doubt putting in the correct passphrase. theres no question about it. I am 10000% putting in the correct passphrase, and its not unlocking. I dont know what else to say. But you cant say its out of your reach, when the least you could do is credit my account with something to make the situation a little better. 570 Dav for me would have been life changing, and for you guys 570 Dav is like pennies.

**Comment by D4rkKr1s** [comment]
Hey jdomsky, did you at least try your passphrase raw before trying to send big amounts? I also used their wallet and I didn't have a single problem sending and withdrawing my 38 Davorcoins. I almost made a typo in the passphrase but then I re-checked it, you could try writing it down and changing maybe 1 letter in each word or maybe seeing if you put an extra space at the start, middle or end. I know you're in panic but maybe you can fix it calmly by trial and error. I hope you get your coins safe and sound!

**Comment by jdomsky** [comment]
Hey D4, thanks so much for your comment man....I really do appreciate you even showing some interest, and trying to help!!! Its crazy that I have got absolutely no help at all from Davor. Yeah when I created my passphrase, you have to enter it in 2 times, so they match, before it creates your passphrase. So im sure that what I created is what im putting in. I wrote it down, I also remember it. and again, ive been working on computers for 15 years, ive been in crypto for 2 years, and ive used many many wallets. I know for certain im putting in the correct passphrase. It wont unlock......and my coins are just stuck in there. basically will go to waste now......sucks bigtime man....like literally to the point where ive been sick. Thanks again for your comment though!!

**Comment by D4rkKr1s** [comment]
No problem man! I've also been using computers for over a decade, but you know, since I am a human I know I can once in a while mess up. I really recommend you to trial and error before you give up. I wish Davor would just give you access to your wallet or if there was a way to bypass the passphrase.

**Comment by jdomsky** [comment]
I understand that even being experienced we can make mistakes. But im not, I wrote it down when I created it. And I clearly remember it. Thats why im so upset, because im going to lose these coins over something that is not my fault. I know for fact I am entering in the right password. And I wish there was something they would do too........but they arent. I mean 570 dav for me would have been life changing. But 570 dav to them is like pennies......
